src/bsonjs.c

```diff
@@ -17,9 +17,7 @@
 #include "bsonjs.h"
 #include <bson.h>
 
-#if PY_MAJOR_VERSION >= 3
 #define PyString_FromString PyUnicode_FromString
```

Comment:
I suggest removing this line and replacing PyString_FromString with PyUnicode_FromString.

Reply:
Done.